type-definitions/immutable.d.ts

```diff
@@ -125,6 +125,23 @@ declare namespace Immutable {
     : // other case : should be kept as is
       T;
 
+  /**
+   * Describes which item in a pair should be placed first when sorting
```

Comment:
```suggestion
   * Describes which item in a pair should be placed first when sorting
   *
   * @ignore
```

to disable automatic output from in the documentation